src/Entity/DatasetSubmission.php

```diff
@@ -1584,7 +1581,7 @@ public function getDatasetFileTransferType() : ?string
      */
     public function setDatasetFileUri(?string $datasetFileUri)
     {
-        $this->datasetFileUri = $datasetFileUri;
+        $this->fileset->getFiles()->first()->setFilePath($datasetFileUri);
```

Comment:
Should i revert the changes made to datasetFileUri setters and getters?

Reply:
yes, they should stay as they are.